build.gradle

```diff
@@ -182,6 +182,7 @@ dependencies {
 
   implementation group: 'com.azure', name: 'azure-data-tables', version: '12.1.2'
 
+  implementation 'com.networknt:json-schema-validator:1.0.42'
```

Comment:
I think we should upgrade to the latest version of this (looks like it's 1.0.64)

Reply:
done